src/app.ts

```diff
@@ -1,6 +1,73 @@
+type ResultData = {
+  id: string;
+  gender: string;
+  age: number;
+  row: number;
+};
+
+type PagingData = {
+  next?: string;
+  previous?: string;
+};
+
+let previousBtn: HTMLElement | any = document.querySelector(
+  "button[data-prevbtn]"
+);
+let nextBtn: HTMLElement | any = document.querySelector(
+  "button[data-nextbtn]"
+);
+let pageView: HTMLElement | any = document.querySelector(
+  "label[data-pageview]"
+);
+let currentPage: number = 1;
+
+const loadData = async (page: number = 1) => {
+  const response = await fetch(
+    `https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84&page=${page}`
+  );
+
+  if (response.status === 200) {
+    const resData = await response.json();
+    const paging: PagingData = resData?.results[0].paging;
+    const data: ResultData[] = resData?.results[0][`${currentPage}`];
+    const pageNo = resData?.info.page;
+    pageView.textContent = `Page: ${pageNo}`;
+    if (paging.previous) {
+      previousBtn.disabled = false;
+    } else {
+      previousBtn.disabled = true;
+    }
+
+    const tableRow = document.querySelectorAll(`tbody[data-sink] > tr`);
+    console.log('tableRow', tableRow[0].children)
```

Comment:
Codacy has a fix for the issue: [Replace `'tableRow',·tableRow[0].children)` with `"tableRow",·tableRow[0].children);`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
    console.log("tableRow", tableRow[0].children);
```